tests/cpptests/test_cpp_circularBuffer.cpp

```diff
@@ -0,0 +1,205 @@
+#include "gtest/gtest.h"
+#include "src/util/circular_buffer.h"
+
+class CircularBufferTest : public ::testing::Test {};
+
+TEST_F(CircularBufferTest, testEmpty) {
+    CircularBuffer buff = CircularBuffer_New(sizeof(int), 16);
```

Comment:
Indentation in this file seems off.

Reply:
Which part of it?

Reply:
Interesting the default tab operation is different than in other files (in other cpp test files as well btw).
Will fix